libs/designsystem/src/lib/components/dropdown/dropdown.component.scss

```diff
@@ -121,6 +121,14 @@ kirby-card {
   }
 }
 
+:host(.with-popover) {
+  &.is-open {
+    kirby-card {
+      display: block;
+    }
+  }
+}
+
```

Comment:
I think we should leave this as is. 

I've been playing around with refactoring `dropdown.component.scss` a bit and i think there's enough to be considered, that it should happen in a separate issue. 

For example: 
- why do we have the classes `is-opening` & `is-open` which we have to track the state of in the controller. Can we get rid of them and simplify it?
- Why do we control opacity? 
- We should find a way to invert the relationship to `kirby-calendar` such that the necessary styles are set in the calendar stylesheet instead of in this file. It gets very hard to see where rules come from when using `host-context` unless strictly necessary. 
- General cleanup; the rule on line 44 could might as well be part of the first :host selector.

I think it is a task in of itself to do this properly and refactor this file and beyond the scope of this issue. 

Perhaps leave a comment behind explaining why this rule exists :)

Reply:
Also can we find a way to invert the `:not(.with-popover)` selector? I personally think something that is NOT is harder to read/mentally grasp as opposed to just adding something on top.